ngraph/test/runtime/interpreter/unit_test.manifest

```diff
@@ -54,17 +54,12 @@ INTERPRETER.reduce_max_3d_to_scalar_double
 INTERPRETER.reduce_max_keep_to_scalar_int8
 INTERPRETER.reduce_max_keep_3d_to_scalar_double
 INTERPRETER.product_to_scalar_int8
-INTERPRETER.max_pool_uint8
-INTERPRETER.max_pool_int8
 INTERPRETER.numeric_double_nan
 INTERPRETER.numeric_double_inf
 INTERPRETER.min_to_scalar_int8
 INTERPRETER.max_trivial_int8
 INTERPRETER.max_to_scalar_int8
 INTERPRETER.max_3d_to_scalar_double
-INTERPRETER.gelu_f64
-INTERPRETER.gelu_backprop_factor_f64
```

Comment:
Can you please comment regarding this change?

Reply:
These tests are passed now